docs/sdk/auth/fragments/ios/federated-identities.md

```diff
@@ -226,12 +226,12 @@ AWSMobileClient.default().federatedSignIn(providerName: IdentityProvider.develop
 	  target 'YOUR-APP-NAME' do
 	    use_frameworks!
 
-	    pod 'AWSFacebookSignIn', '~> 2.12.0'     # Add this new dependency
-	    pod 'AWSAuthUI', '~> 2.12.0'             # Add this dependency if you have not already added
+	    pod 'AWSFacebookSignIn', '~> 2.13.0'     # Add this new dependency
+	    pod 'AWSAuthUI', '~> 2.13.0'             # Add this dependency if you have not already added
 	    
 	    # Other Pod entries
-	    pod 'AWSMobileClient', '~> 2.12.0'
-	    pod 'AWSUserPoolsSignIn', '~> 2.12.0'
+	    pod 'AWSMobileClient', '~> 2.13.0'
+	    pod 'AWSUserPoolsSignIn', '~> 2.13.0'
```

Comment:
On https://aws-amplify.github.io/docs/sdk/ios/authentication#facebook-login-in-your-mobile-app these are listed as `2.12.0`. Is the idea to bump all references to `2.13.0`?

```ruby
 platform :ios, '9.0'
   target 'YOUR-APP-NAME' do
     use_frameworks!

     pod 'AWSFacebookSignIn', '~> 2.12.0'     # Add this new dependency
     pod 'AWSAuthUI', '~> 2.12.0'             # Add this dependency if you have not already added
	    
     # Other Pod entries
     pod 'AWSMobileClient', '~> 2.12.0'
     pod 'AWSUserPoolsSignIn', '~> 2.12.0'
	    
   end
```

Reply:
yes, on iOS side, latest version is 2.13.x. I believe i did the upgrade on the master branch (partially though). this PR on v2-preview and https://github.com/aws-amplify/docs/pull/1452 on master should get everything aligned

Reply:
Thanks for taking care of the master branch as well